### ISSUE TITLE
8.2.12. New - Screen Name/Title: Create a service – Missing aria-required="true" is redundant

### ISSUE THREAD

**Issue by jzbahrai** [cds-snc/notification-planning#2460]
Component: Create template - Will you send the message by email or text?
Observation:
The <fieldset> uses aria-required="true", but this attribute is not supported on fieldset by most screen readers. HTML native validation or ARIA at the radio group level (or individual inputs) is more reliable.

(Attachment: 8.2.12. Missing aria-required="true" is redundant.png)

Recommendation: 
Remove aria-required="true" from the <fieldset> and ensure client-side validation is clearly announced. If required, move aria-required="true" to the radio buttons or handle with proper validation fee
